src/styles/h5p-multi-media-choice.scss

```diff
@@ -161,4 +161,130 @@
     white-space: nowrap;
     width: 1px;
   }
+
+  .h5p-multi-media-content-media-button-video{
+    font-family: 'H5PFontAwesome4';
+    position: absolute;
+    background-color: crimson;
+    z-index: 3;
+    bottom: 0;
+    right: 0;
+    margin: 0.5em;
+    width: 3em;
+    height: 3em;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2),0 6px 20px 0 rgba(0,0,0,0.19);
```

Comment:
`box-shadow: h-offset v-offset blur spread color inset;` something is weird in your line it looks like you have double times the property ?